sdcm/results_analyze/__init__.py

```diff
@@ -222,15 +232,21 @@ def check_regression(self, test_id, data, is_gce=False):  # pylint: disable=too-
             build_id = ''
 
         last_events, events_summary = self.get_events(event_severity=[
-            Severity.CRITICAL.name, Severity.ERROR.name, Severity.WARNING.name])
-        reactor_stall_events, reactor_stall_events_summary = self.get_debug_events()
+            Severity.CRITICAL.name, Severity.ERROR.name])
+        debug_events, _ = self.get_debug_events()
```

Comment:
this var is not used - pylint throws error.
btw. do we need to limit debug events too?

Reply:
you are right, i missed that, as the moving parts using it were moved to the new functions...
removing it.

no, at this point we don't want to remove it... in this PR we are removing ALL kernel callstack events, and all left DEBUG events are show in a file (and not in the body or something).

so in a sense, it is already being limited by removing part of these events.